src/JsonSchema/Data/StockMarketSchema.ts

```diff
@@ -25,9 +23,18 @@ export const StockMarketSchema = {
   $schema: "http://json-schema.org/draft-07/schema#",
   type: "object",
   patternProperties: {
-    "^(?!.*(lastUpdate|Orders|storedCycles|ticksUntilCycle))": {
+    /**
+     * IStockMarket is an intersection type combining:
+     * - Record<string, Stock> [1]
+     * - {lastUpdate: number;Orders: IOrderBook;storedCycles: number;ticksUntilCycle: number;} [2]
+     *
+     * StockMarketSchema contains:
+     * - patternProperties: Defines [1]. The following regex matches all properties that are not in [2]. It defines the
+     * map of "Full stock name -> Stock".
+     * - properties: Define [2].
+     */
```

Comment:
I would add (here or somewhere more appropriate) a result of your investigation into the keys, specifically that having extras is tolerated (and thus adding new ones shouldn't require a save migration).